docs/source/docs/level-up/basic_skills/03_dataset_import_export.rst

```diff
@@ -0,0 +1,89 @@
+=============
+Level 3: Data Import and Export
+=============
+
+Datumaro is a tool that supports public data formats across a wide range of tasks such as
+classification, detection, segmentation, pose estimation, or visual tracking.
+To facilitate this, Datumaro provides assistance with data import and export via both Python API and CLI.
+This makes it easier for users to work with various data formats using Datumaro.
+
+Prepare dataset
+============
+
+For the segmentation task, we here introduce the Cityscapes, which collects road scenes from 50
+different cities and contains 5K fine-grained pixel-level annotations and 20K coarse annotations.
+More detailed description is given by :ref:`here <Cityscapes>`.
+The Cityscapes dataset is available for free `download <https://www.cityscapes-dataset.com/downloads/>`_.
```

Comment:
It would be good to show how to prepare the dataset on the shell. For example, we can demonstrate the dataset download, extract, and directory layout.

```console
$ curl https://www.cityscapes-dataset.com/downloads/.... -o dataset.zip
$ unzip dataset.zip
$ ls dataset
annotation images
```

Reply:
I am sorry that Cityscapes is required to log in to the system for free download.